spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    let(:attr_valid) { attributes_for(:book_suggestion) }
+    let(:attr_invalid) { attributes_for(:book_suggestion, year: nil) }
+
+    subject(:http_request_valid) do
```

Comment:
Usually, the subject is beefore the lets, and remember: Just one subject per describe, in this case the `POST #create` describe should have only have subject, and you change the params in every context with lets